src/ui/pages/UserInitialPage/InitialPage.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate, useParams } from "react-router-dom";
+import { adminService } from "../../../services";
+import { useEffect } from "react";
+import { admins } from "../../../core/admin";
+
+const InitialPage = () => {
+  const { email } = useParams();
+  const redirect = useNavigate();
+  console.log(email);
+
+  useEffect(() => {
+    const initAdmin = async () => {
+      if (email) {
+        const data = await adminService.initAdmin(email);
+        if (data.isSuper) {
+          redirect("/verifyquestion");
+        }
+        const isAdmin = admins.includes(email);
```

Comment:
why do you have this check here? use the data to check if user is super or not.

Reply:
this is a check for just admins